src/muya/lib/parser/marked/lexer.js

```diff
@@ -207,11 +207,11 @@ Lexer.prototype.token = function (src, top) {
       src = src.substring(cap[0].length)
       bull = cap[2]
       let isOrdered = bull.length > 1 && /\d{1,9}/.test(bull)
-
+      console.log(bull.length, cap)
       this.tokens.push({
         type: 'list_start',
         ordered: isOrdered,
-        listType: bull.length > 1 ? (/\d{1,9}/.test(bull) ? 'order' : 'task') : 'bullet',
+        listType: bull.length > 1 ? 'order' : (/^( {0,3})([-*+]) \[([xX]{1}| )\]/.test(cap[0]) ? 'task' : 'bullet'),
```

Comment:
Why not just `/^( {0,3})([-*+]) \[([ xX])\]/`?

Reply:
Task list items are not clearly defined. According to GFM a task list item can be `- [ ]` but GitHub renders a task list item only if `- [ ] a`. Maybe we should check for `/^( {0,3})([-*+]) \[([ xX])\] /` like all popular parsers?

Reply:
What's the meaning `just`?

Reply:
My fault!